### DISCUSSION TITLE
I can't start the server

### DISCUSSION THREAD

**Discussion by Jonck-lens** [Melledy/Grasscutter#2371] [Q&A]
I waited a long time after I executed the command "java-jar grasscutter" and the program only returned the following.
![屏幕截图 2023-09-16 205552](https://github.com/Grasscutters/Grasscutter/assets/136221840/019b1a9a-d466-4311-979a-0afe2207e2cb)
In the past, programs didn't get stuck here. I've tried everything I know, but to no avail.

Thank you for your help. By the way, my English is not very good

**Comment by Jonck-lens** [comment]
about:
  system:   windows 11 22h2
  java:   java version "17.0.8" 2023-07-18 LTS
            Java(TM) SE Runtime Environment (build 17.0.8+9-LTS-211)
            Java HotSpot(TM) 64-Bit Server VM (build 17.0.8+9-LTS-211, mixed mode, sharing)
grasscutter:  1.7.1
resource：4.0
jar File source: precompiled

**Comment by Rush-525** [comment]
你需要data文件，建议你用grassclipper/cultivation启动器重新下载数据

**Reply by Rush-525** [reply]
帮你上传了data文件（只需管下面这个，1文件不用管）
https://github.com/Rush-525/Grasscutter/tree/data/data/gacha
在这个仓库里面下载mappings.js  放到对应目录里面即：data/gacha/mapping.js

**Reply by Rush-525** [reply]
By the way，I not be good at English

**Reply by NameLord** [reply]
By the way its NOT i can't start the server.. its HOW you can't start one ?
